common/common.go

```diff
@@ -319,8 +336,14 @@ func dockerBuild(verbose bool, fpath string, ff *FuncFile, buildArgs []string, n
 	return nil
 }
 
-func dockerBuildV20180708(verbose bool, fpath string, ff *FuncFileV20180708, buildArgs []string, noCache bool) error {
-	err := dockerVersionCheck()
+func containerEngineBuildV20180708(verbose bool, fpath string, ff *FuncFileV20180708, buildArgs []string, noCache bool) error {
+	containerEngineType, err := GetContainerEngineType()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Using Container engine ", containerEngineType)
```

Comment:
More like an OCD 😄 commas between `Println` add spaced automatically. I hope all these will be cleaned up before merging.